### ISSUE TITLE
Replacements Error Messages

### ISSUE THREAD

**Issue by gauntface** [eslint/eslint#4927]
Hi,

I'm constantly getting the following error messages in eslint form the command line:
```sh
$ eslint -c .eslintrc ./src/*.js

/project/path/src/filename.js
  14:1  error  Rule 'no-empty-class' was removed and replaced by: no-empty-character-class  no-empty-class
  14:1  error  Rule 'no-extra-strict' was removed and replaced by: strict                   no-extra-strict
  14:1  error  Rule 'no-reserved-keys' was removed and replaced by: quote-props             no-reserved-keys
  14:1  error  Rule 'no-space-before-semi' was removed and replaced by: semi-spacing        no-space-before-semi
  14:1  error  Rule 'no-wrap-func' was removed and replaced by: no-extra-parens             no-wrap-func
```
I've searched my eslint file and it's not defining any of those rules and the only way I've found that makes these errors go away is to define them and disable the warning i.e. add 'no-empty-class': 0 to my eslint rule.

I've tried wiping my npm modules with no changes. I'm pretty much out of ideas where this problem comes from.

Reducing the eslint file down to:

    {
      "env": {
        "es6": true,
        "browser": true
      }
    }

Still produces errors for the rule changes. I'd really appreciate any advice or fixes for this.

**Comment by eslintbot** [comment]
@gauntface Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gauntface** [comment]
$ eslint -v
v1.10.3

**Comment by gyandeeps** [comment]
Maybe you have a `.eslintrc` file up the chain in folder hierarchy and that getting pulled.
Best way to know this is to run using `--debug` flag to see whaere are all the configurations coming from.

```sh
$ eslint --debug -c .eslintrc ./src/*.js
```

**Comment by gauntface** [comment]
That was exactly it - thank you so much - it was driving me insance.
